Dockerfile

```diff
@@ -1,66 +1,37 @@
-FROM golang:1.11 as builder
+FROM golang:1.13.1 as builder
 
-# Set up workdir
+# Set up workdir to make go mod work
 WORKDIR /cayley
 
-# Restore dependencies
+# Install dependencies first
 COPY go.mod go.sum ./
 RUN go mod download
 
-# This will be used to init cayley and as config file in the final image.
-# Make sure you start every path with %PREFIX% to make it available in both 
-# the builder image and the final image.
-RUN echo '{"store":{"backend":"bolt","address":"%PREFIX%/data/cayley.db"}}' > config.json
-
 # Create filesystem for minimal image
-RUN mkdir -p /fs/assets
-RUN mkdir -p /fs/bin
-RUN mkdir -p /fs/data
-RUN mkdir -p /fs/etc
-RUN mkdir -p /fs/tmp
-RUN sed 's_%PREFIX%__g' config.json > /fs/etc/cayley.json
-
-ENV PATH /fs/bin:$PATH
-
-# Copy CA certs from builder image to the filesystem of the cayley image
-RUN mkdir -p /fs/etc/ssl/certs
-RUN cp /etc/ssl/certs/ca-certificates.crt /fs/etc/ssl/certs/ca-certificates.crt
-
-RUN mkdir -p /fs/lib/x86_64-linux-gnu
-RUN cp /lib/x86_64-linux-gnu/libc-* /fs/lib/x86_64-linux-gnu/
-
-# Add assets to target fs
-COPY docs /fs/assets/docs
-COPY static /fs/assets/static
-COPY templates /fs/assets/templates
-
-# Add and build static linked version of cayley
-# This will show warnings that glibc is required at runtime which can be ignored
-COPY . .
-RUN go build \
-  -ldflags="-linkmode external -extldflags -static -X github.com/cayleygraph/cayley/version.GitHash=$(git rev-parse HEAD | cut -c1-12)" \
-  -a \
-  -installsuffix cgo \
-  -o /fs/bin/cayley \
-  -v \
-  ./cmd/cayley
-
-RUN sed 's_%PREFIX%_/fs_g' config.json > /etc/cayley.json
-RUN cayley init --config /etc/cayley.json
-
+RUN mkdir -p /fs/etc/ssl/certs /fs/lib/x86_64-linux-gnu /fs/tmp /fs/bin /fs/assets; \
+    # Copy CA Certificates
+    cp /etc/ssl/certs/ca-certificates.crt /fs/etc/ssl/certs/ca-certificates.crt; \
+    # Copy C standard library
+    cp /lib/x86_64-linux-gnu/libc-* /fs/lib/x86_64-linux-gnu/
+
+ADD . .
+
+# Move assets into the filesystem
+RUN mv docs static templates /fs/assets
+
+RUN SHORT_SHA=$(git rev-parse --short=12 HEAD); \
+    go build \
+    -ldflags="-linkmode external -extldflags -static -X github.com/cayleygraph/cayley/version.GitHash=$SHORT_SHA" \
+    -a \
+    -installsuffix cgo \
+    -o /fs/bin/cayley \
+    -v \
+    ./cmd/cayley
 
 FROM scratch
-LABEL maintainer="Yannic Bonenberger" \
-           email="contact@yannic-bonenberger.com"
 
-# Expose the port and volume for configuration and data persistence. If you're
-# using a backend like bolt, make sure the file is saved to this directory.
 COPY --from=builder /fs /
-VOLUME ["/data"]
 
 EXPOSE 64210
 
-# Adding everything to entrypoint allows us to init+load+serve
```

Comment:
This comment was useful as well :)

Also, please preserve line break before the EOF. Some tools require them.